feedback.md

```diff
@@ -0,0 +1,21 @@
+# Word-Guess Game
+## What We're Looking For
+
+|  Feature 	|   Feedback	&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;|
+|---	|---	|
+|   Both Teammates contributed to the codebase	|   	|
+|   Created a Class to encapsulate game functionality	|   	|
+|   Used methods to DRY up your code	|   	|
+|   Created instance variables & local variables where appropriate	|   	|
+|   Used Arrays to store lists of letters guessed	|   	|
+|   Used variables & random numbers to allow the game to function with multiple words, no hard-coded answers.	|   	|
+|   Programmed "defensively" to detect errors in user input.	|   	|
+|   Created interesting ASCII Art	|   	|
+|   **Extras**	|   	|
+|   Used the Colorize Gem	|   	|
+|   User is not penalized for guessing the same word twice	|   	|
+|   Multiple difficulty levels	|   	|
+|   The program can accept the whole word in one guess	|   	|
+|  Regular Commits with meaningful commit messages. 	|   	|
```

Comment:
I'd disagree that these three would be considered *extras*. Maybe we just have a *baseline* section that we have with these three in every single project submission